opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
In the original implementation ``m_mask`` is kind of filter, it can decide which specific type of message can be logged or not, which is good. The new implementation of the ``m_mask`` is a weaker one, which decides some types of the messages not only one type can go into OpmLog.